### ISSUE TITLE
About the pre-process of val set

### ISSUE THREAD

**Issue by 1170300714** [wenxi-yue/SurgicalSAM#17]
Thanks for your great work!

I am referring to your preprocessing code to handle my private dataset. However, it seems that the data_preprocess.py only implements preprocessing for the training set. Could you please release the preprocessing code for the test set? Thank you!

**Comment by wenxi-yue** [comment]
Hi,

Thank you for your interest.

The preprocessing steps for the training set and the validation set are quite similar. The only difference is that for the validation set, we do not apply data augmentation. For processing the validation set, we use the original frames and original binary annotation masks (rather than the frames and binary annotation masks after augmentation) to compute the SAM features and class embeddings.

So you can refer to the code for computing the SAM features [here](https://github.com/wenxi-yue/SurgicalSAM/blob/bf5d3a8dacc3e49a7f324186da8bf0afbecc249f/surgicalSAM/tools/data_preprocess.py#L215-L218), and the code on computing class embeddings [here](https://github.com/wenxi-yue/SurgicalSAM/blob/bf5d3a8dacc3e49a7f324186da8bf0afbecc249f/surgicalSAM/tools/data_preprocess.py#L237-L250). In other words, you can use `data_preprocess.py`, but remove all parts related to data augmentation for processing the validation set.
